lib/Attribute.js

```diff
@@ -42,6 +42,9 @@ function Attribute(schema, name, value) {
   if (this.type.name === 'map'){
 
     if(value.type) {
+      if (!value.map) {
+        throw new errors.SchemaError('No map schema given for attribute:' + this.name );
```

Comment:
```suggestion
        throw new errors.SchemaError(`No map schema given for attribute: ${this.name}`);
```

Super picky, I know. But PR #465 is doing this, so I think from now on we should do it throughout the code.